README.md

```diff
@@ -53,46 +53,37 @@ const customStyles = {
 // Make sure to bind modal to your appElement (http://reactcommunity.org/react-modal/accessibility/)
 Modal.setAppElement('#yourAppElement')
 
-class App extends React.Component {
-  constructor() {
-    super();
-
-    this.state = {
-      modalIsOpen: false
-    };
-
-    this.openModal = this.openModal.bind(this);
-    this.afterOpenModal = this.afterOpenModal.bind(this);
-    this.closeModal = this.closeModal.bind(this);
+function App(){
+  openModal = openModal.bind(this);
+  afterOpenModal = afterOpenModal.bind(this);
+  closeModal = closeModal.bind(this);
```

Comment:
I think you don't need to `bind` in this case. They can be removed.

Reply:
@diasbruno hm, you are right.
I forget remove it.